src/patterns/language-selection/index.njk

```diff
@@ -0,0 +1,42 @@
+---
+title: Change Language
+group: Help users to...
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+
+## When to use this pattern
+
+This pattern should be if you service is available in multiple languages.
+
+## How to use this pattern
+
+Language selection is achieved by rendering a language selector component in the header element, and a list of available lanauges in the footer. The header lanauge selector is for viewports larger than mobile, and the footer language selector is for viewports smaller than tablet.
```

Comment:
```suggestion
Language selection is achieved by rendering a language selector component in the header element, and a list of available languages in the footer. The header language selector is for viewports larger than mobile, and the footer language selector is for viewports smaller than tablet.
```
...but perhaps this shouldn't refer to a 'component' unless we decide to create it as a component.

Reply:
So I did originally have a page documenting the language selector component, but it was strange because that only really takes care of the desktop switcher, there isn't a component for the footer language selector. The language selector dropdown should only ever show in the header, and is imported by the header so I'm not really sure whether we should be mentioning it as a component, as really it's a feature of the header. It only has a component folder to stop the header component from getting too complex.

Reply:
Maybe it could be documented as a variant of the header, which we could reference here?  or a variant of the select component?

Reply:
@will-madebymoon Should we align the styles of these? 
<img width="386" alt="screenshot 2019-02-01 at 10 09 29" src="https://user-images.githubusercontent.com/43346934/52116770-ec263980-2609-11e9-84d4-9b579a9ede2b.png">
<img width="178" alt="screenshot 2019-02-01 at 10 09 14" src="https://user-images.githubusercontent.com/43346934/52116771-ec263980-2609-11e9-9fa5-409bdab2187c.png">

Reply:
Agreed.

Reply:
Whilst it isn't a select component it should be consistent.

Reply:
Not sure how it ended up light grey with a blue arrow
<img width="177" alt="screen shot 2019-02-01 at 11 15 56" src="https://user-images.githubusercontent.com/8184622/52119975-d9643280-2612-11e9-9416-db5ca5ca976d.png">

Reply:
It should have the same states as these https://deploy-preview-110--ons-design-system.netlify.com/components/select/

Reply:
I've now updated the styling to match the select component https://deploy-preview-110--ons-design-system.netlify.com/patterns/language-selection/examples/many-languages/. @will-madebymoon are the menu item heights ok? I've made them match the height of the "button" that opens the menu. Only thing is that it makes the white part of the header taller than it would be without the dropdown variant of the language selector - 65px vs 49px.

Reply:
After discussion with @will-madebymoon I've dropped the dropdown version of the language selector's height down to 36px to match the sketch file, and adjusted the header css to prevent it from getting taller when the dropdown variant is added.